Content.Client/CrewMedal/UI/CrewMedalWindow.xaml

```diff
@@ -0,0 +1,13 @@
+<DefaultWindow xmlns="https://spacestation14.io"
+            xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+            Title="{Loc 'crew-medal-ui-header'}">
+    <BoxContainer Orientation="Vertical" SeparationOverride="4" MinWidth="150">
+        <Label Text="{Loc 'crew-medal-ui-reason'}" />
+        <BoxContainer Orientation="Horizontal">
+            <LineEdit Name="ReasonLineEdit" HorizontalExpand="True" />
```

Comment:
Opportunity for a little UI nicety in here:

Add a Placeholder="{Loc crew-medal-reason-placeholder}" and corresponding translation? Or maybe populate the placeholder from the entity description?